### PR TITLE
Add all the remaining testimonials from quota increase requests

### LINKED ISSUE #688: Feedback from users that requested quota increase

### PR DESCRIPTION
**Pull request by beatrizserrano**
fix #688

### CODE REVIEW

File: _posts/2021-08-06-use-case-romain-garrigues.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'UseGalaxy.eu Use Case: Plant research'
+tags: [UseCase]
+---
+
+[__Romain Garrigues__](https://www.uantwerpen.be/en/staff/romain-garrigues/) is doing a PhD on plant research at the [Department of Biology, University of Antwerp (Belgium)](https://www.uantwerpen.be/en/departments/biology/). Thank you Romain!
+
+
+> Galaxy is useful for my work because I am doing metadata analysis of RNA sequencing based on the different climatic conditions and different plant models. With the same parameters, I will be able to continue to work and achieve step by steps goal of my PhD.
```
Comment (bgruening) [inline _posts/2021-08-06-use-case-romain-garrigues.md:10]: too less for a blog post?

**Review by bgruening — COMMENTED** [review]
Thanks.

File: _posts/2021-08-06-use-case-romain-garrigues.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2021-08-06-use-case-romain-garrigues.md:10]: do you think we should remove it?

**Comment by bgruening** [comment]
Danke!

**Pull request merged**